src/ray/util/event.h

```diff
@@ -217,15 +225,29 @@ class RayEvent {
 
   const RayEvent &operator=(const RayEvent &event) = delete;
 
+  // Only for test
+  static void SetLevel(const std::string &event_level);
+
+  FRIEND_TEST(EVENT_TEST, TEST_LOG_LEVEL);
+
  private:
   rpc::Event_Severity severity_;
   std::string label_;
   json custom_fields_;
   std::ostringstream osstream_;
 };
 
+/// Ray Event initialization.
+///
+/// This function is not thread-safe. We should use this function when main thread starts.
```

Comment:
consider using https://abseil.io/docs/cpp/guides/synchronization#call_once to ensure this is only called once.

Reply:
+1. Or make some singleton logic (e.g., have a variable like "initialized" and if initialized=true, crash ray)